feat(doc): add steps to verify lxd host support for vms (#15228)
canonical/lxd: doc/tutorial/first_steps.md

   This will create a minimal setup with default options.
   If you want to tune the initialization options, see {ref}`initialize` for more information.

> 1. LXD provides support for {ref}`containers-and-vms`. However, LXD support for virtual machines depends on the host operating system support for virtualization. Below steps verify support for virtual machine instances.

Reviewer: ```suggestion
1. LXD supports both {ref}`containers-and-vms`. For virtual machines, your host system must be capable of KVM virtualization. To check this, run the following command:
```

>        sudo apt install cpu-checker

Reviewer: ````suggestion
```
lxc info | grep -FA2 'instance_types'
```
````

>        sudo kvm-ok

Reviewer: ```suggestion
```

>    Below output is expected if your systems supports virtualization.

Reviewer: ```suggestion
   The following output indicates that your host system is capable of virtualization:
```

>        INFO: /dev/kvm exists

Reviewer: ````suggestion
```
  instance_types:
  - container
  - virtual-machine
```
````

>        KVM acceleration can be used

Reviewer: ```suggestion
```

>    Below output indicates your system doesn't support virtualization. In this scenario LXD will provide support only for containers.

Reviewer: ```suggestion
   If `virtual-machine` fails to appear in the output, this indicates that the host system is not capable of virtualization. In this case, LXD can only be used for containers. You can proceed with this tutorial to learn about using containers, but disregard the steps that refer to virtual machines.
```

>        INFO: Your CPU does not support KVM extensions

Reviewer: ```suggestion
```

>        KVM acceleration can NOT be used

Reviewer: ```suggestion
```

   This will create a minimal setup with default options.
   If you want to tune the initialization options, see {ref}`initialize` for more information.

1. LXD supports both {ref}`containers-and-vms`. For LXD virtual machines, your host system must be capable of KVM virtualization. To check this, run the following command:

```

> lxc info | grep -FA2 'instance_types'

Reviewer: ```suggestion
   lxc info | grep -FA2 'instance_types'
```

Author: Committed separately and squashed into single commit.

> ```

Reviewer: ```suggestion
   ```
```

   The following output indicates that your host system is capable of virtualization:

> ```

Reviewer: ```suggestion
   ```
```

  instance_types:
  - container
  - virtual-machine

> ```

Reviewer: ```suggestion
   ```
```
